aiohttp/web_request.py

```diff
@@ -510,8 +510,19 @@ def text(self):
         return bytes_body.decode(encoding)
 
     @asyncio.coroutine
-    def json(self, *, loads=json.loads):
-        """Return BODY as JSON."""
+    def json(self, *, loads=json.loads, wrong_format=None, if_check=True):
+        """
+        Return BODY as JSON.
+        If the `Content-Type` is not `application/json`,
+        then the request is rejected.
+        You can also handle it manually by passing the
+        `wrong_format` parameter.
+        """
+        if if_check and self.content_type != 'application/json':
```

Comment:
How would I use, say, [jsonapi](http://jsonapi.org/) if it will never match this hardcoded content-type?